index.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="media-style.css" />
+    <title>Anicet's Capstone Project</title>
+</head>
+
+<body>
+    <div class="head-line">
+        <img src="./images/facebook.png" alt="">
+        <img src="./images/twitter.png" alt="">
+        <div>English</div>
+        <div>My page</div>
+        <div>Logout</div>
+    </div>
+    <header>
+        <div class="logo">
+            <a href=""><img src="./images/NFTSbck.jpg" alt="This a NFT logo" /></a>
+        </div>
+
+        <nav>
+            <ul>
+                <li><a href="About.html">About</a></li>
+                <li><a href="">Progran</a></li>
+                <li><a href="">Join</a></li>
+                <li><a href="">Sponsors</a></li>
+                <li><a href="">News</a></li>
+                <li><a href="">Campaign</a></li>
```

Comment:
- [ ] I could see we improved the look of the campaign button, I would just suggest using text color according to Behance to match the design currently it's black let's kindly change it to proper color
![image](https://user-images.githubusercontent.com/87689547/159018577-21768609-eeca-4058-b6f7-31b3064ff6bf.png)